_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -32,7 +32,12 @@
         "drupal/simple_sitemap": "2.3.0",
 	"drupal/pathauto": "dev-1.x",
         "drupal/config_update": "1.1.0",
-        "drupal/ctools": "3.0.0-alpha26"
+        "drupal/ctools": "3.0.0-alpha26",
+        "drupal/entity_reference_revisions": "dev-1.x",
+        "drupal/paragraphs": "dev-1.x",
+        "drupal/subpathauto": "dev-1.x",
+        "drupal/field_collection": "dev-1.x",
```

Comment:
8.x-1.0 is final, but 8.x-3.x-dev is where work is happening, not sure what we want to do here. 3.x has integration with Entity Reference which may make more sense.